css-transitions-2/Overview.bs

```diff
@@ -84,11 +84,15 @@ the transition was generated (e.g. 'border-left-width').
 ISSUE: Update the defining instance of [=transitionable=]
 once it is ported from Level 1.
 
-In CSS Transitions Level 2,
-property values are [=transitionable=]
-<em>unless</em> they have an [=animation type=] that is [=not animatable=].
-Values with a [=discrete=] [=animation type=] <em>are</em> [=transitionable=],
-and flip at 50% progress (<var>p</var> = 0.5).
```

Comment:
I think it's worth adding this back as a note.  In bikeshed that should be doable by making it its own paragraph beginning with "Note: ", though I think in full it probably needs to be something like: "Note: When values with a [=discrete=] [=animation type] are transitioned, they flip at 50% progress."

Reply:
done, thanks!